cSpell.json

```diff
@@ -0,0 +1,108 @@
+{
+  "version": "0.2",
+  "language": "en",
+  "words": [
```

Comment:
ℹ️ NOTE: `words` in this file will be suggested alongside spellcheck in IDEs that make use of cSpell (e.g. VS Code). `ignoreWords` will not. I make a best-guess effort to separate the two.